apps/opik-frontend/src/components/pages/DatasetsPage/DatasetsActionsPanel.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useCallback, useRef, useState } from "react";
+import { Trash } from "lucide-react";
+
+import { Button } from "@/components/ui/button";
+import { Dataset } from "@/types/datasets";
+import useDatasetBatchDeleteMutation from "@/api/datasets/useDatasetBatchDeleteMutation";
+import ConfirmDialog from "@/components/shared/ConfirmDialog/ConfirmDialog";
+import TooltipWrapper from "@/components/shared/TooltipWrapper/TooltipWrapper";
+
+type DatasetsActionsPanelsProps = {
+  datasets: Dataset[];
+};
+
+const DatasetsActionsPanel: React.FunctionComponent<
+  DatasetsActionsPanelsProps
+> = ({ datasets }) => {
+  const resetKeyRef = useRef(0);
+  const [open, setOpen] = useState<boolean>(false);
+  const disabled = !datasets?.length;
+
+  const { mutate } = useDatasetBatchDeleteMutation();
+
+  const deleteDatasetsHandler = useCallback(() => {
+    mutate({
+      ids: datasets.map((d) => d.id),
+    });
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [datasets]);
```

Comment:
i would add `mutate` to the list of dependencies

Reply:
Fixed